src/components/AuthorPreview.jsx

```diff
@@ -14,14 +14,15 @@ const AuthorPreview = ({ authorData }) => {
   }
 
   const parseId = (fullId) => {
+    console.log(["epic", "cool", "sick"].at(-1))
```

Comment:
this line shouldn't be here

Reply:
excellent point team mate. i will make quick work of this!